packages/design-system/src/components/locale/es.json

```diff
@@ -57,6 +57,7 @@
     "lock": "Candado",
     "menu": "Icono de menú",
     "next": "Siguiente",
+    "print": "Imprimir",
```

Comment:
Not sure if you're bilingual. If not, you will probably want to confirm that this word is the correct one for this verb in Spanish (it's kinda like how ["stamp" can have several words in Spanish](https://www.speakinglatino.com/stamp-in-spanish/), just wanna confirm with the language person that this is the correct/commonly used translation).

Reply:
@zarahzachz No, sadly, I don't speak Spanish. I popped this into google translate and got "Imprimir". Is there a language expert at CMS I could reach out to? I'm assuming I need to get Melissa's approval, but it would be nice to have it correct beforehand.

Reply:
I can ask Krithi who the language resource is because I always forget. Also, if you're doing multiple icons, probably best to batch the translation requests for all of them because it could take a little while to get a reply.

You shouldn't need Melissa's approval on this sorta stuff (design should have cleared the icon additions with her before it got assigned to code). At least that's how it worked for us. You'll probably wanna chat with your team to see if that's an expectation.

Reply:
Got a name: Von Portalatin (we've consulted with her for healthcare stuff in the past).

Might be good to specify that you're looking for the Spanish verbs for your icons (maybe that's specific enough for her? 🤷‍♀️)

Reply:
@zarahzachz  Thanks! Was able to confirm with my team that "Imprimir" is correct.

Reply:
Another thing we can do when we need a CMS-approved translation but don't want to make a new request is to inspect product teams' translations or ask them for their translation for a particular pattern.